lib/config.js

```diff
@@ -60,6 +60,7 @@ const Config = function () {
   this.sign_widevine_key = process.env.SIGN_WIDEVINE_KEY || ''
   this.sign_widevine_passwd = process.env.SIGN_WIDEVINE_PASSPHRASE || ''
   this.signature_generator = path.join(this.srcDir, 'third_party', 'widevine', 'scripts', 'signature_generator.py') || ''
+  this.forceComponentBuild = JSON.parse(getNPMConfig(['force_component_build']) || false)
```

Comment:
Should this be a cli arg instead of an environment variable? A developer can run different types of builds in the same environment and may not want to edit the environment every time the different build command is run. Or maybe I'm not understanding the sentiment behind component_build?

Reply:
That makes sense to me. https://github.com/brave/brave-browser/pull/5052 too?